listenbrainz/bigquery-writer/bigquery-writer.py

```diff
@@ -16,6 +15,12 @@
 from listenbrainz.bigquery import NoCredentialsVariableException, NoCredentialsFileException
 from oauth2client.client import GoogleCredentials
 
+from listenbrainz import default_config as config
```

Comment:
Not sure if these imports are worth it or if there is a better way to do this.

CB does it this way but they don't have any config imports outside of Flask apps and AB has a PR https://github.com/metabrainz/acousticbrainz-server/pull/236 replacing `config.py` with `default_config.py` and `custom_config.py`. I would be happy with a finding a consistent way of doing this, so this might be the way.